src/tfvc/scm/model.ts

```diff
@@ -109,15 +111,22 @@ export class Model {
 
     private async update(): Promise<void> {
         const changes: IPendingChange[] = await this._repository.GetStatus();
+        const foundConflicts: IConflict[] = await this._repository.FindConflicts();
+
+        //if (foundConflicts.find(c => c.type === ConflictType.NAME_AND_CONTENT || c.type === ConflictType.RENAME)) {
+        //    TODO: Send telemetry
```

Comment:
Yes, I think that's a good idea.  We should have the ability to send telemetry from wherever we need to.